addons/web/static/tests/views/kanban_view_tests.js

```diff
@@ -4666,6 +4666,80 @@ QUnit.module("Views", (hooks) => {
         }
     );
 
+    QUnit.test("Move record in grouped by date with progress bars", async (assert) => {
```

Comment:
Running this test in debug shows that the values of the `sum_field` in the progress bar are wrong (always 0).
I actually have a fix for this [here](https://github.com/odoo/odoo/pull/98837/commits/4c7356a6f95c84ff2bff3167dd539e855b6d0ff4)